### PR TITLE
[gs][bm] Add GS for vcd

### PR DESCRIPTION
**Pull request by z9r5**
## Description
Add getting started for VMware Cloud Director.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [ ] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: docs
type: chore
summary: Add getting started for VMware Cloud Director.
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/site/_includes/getting_started/vcd/layouts/STANDARD.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+Currently this is the only placement strategy for vSphere provider. It provides deep customization options for lots of network layouts.
+
+Here we'll describe setting up a cluster on a single Network which has a DHCP server and access to the Internet.
```
Comment (kirkonru) [inline docs/site/_includes/getting_started/vcd/layouts/STANDARD.md:3]:
```suggestion
Currently, this is the only layout strategy for the vSphere provider. It features a wide range of customization options for many various network layouts.

Here, we'll cover setting up a cluster on a common Network that has a DHCP server and Internet access.
```

File: ee/candi/cloud-providers/vcd/docs/ENVIRONMENT.md [hunk 2]
```diff
@@ -7,52 +7,218 @@ description: "Configuring VMware Cloud Director for Deckhouse cloud provider ope
...
+* _VApp_
+* _StoragePolicy_
+* _SizingPolicy_
+* _Network_
+* _EdgeRouter_
+* _Catalog_
 
-### Adding network
+Organization, VirtualDataCenter, StoragePolicy, SizingPolicy, EdgeRouter and Catalog must be provided by your VMware Cloud Director service provider.
+Also, in the tenant, you need to grant the following rights to change VM parameters (use the [instruction](https://kb.vmware.com/s/article/92067)):
```
Comment (kirkonru) [inline ee/candi/cloud-providers/vcd/docs/ENVIRONMENT.md:20]:
```suggestion
Organization, VirtualDataCenter, StoragePolicy, SizingPolicy, EdgeRouter, and Catalog must be provided by your VMware Cloud Director service provider.
Also, in the tenant, you will need to grant the following rights to able to modify the VM parameters (refer to the [instruction](https://kb.vmware.com/s/article/92067)):
```

File: ee/candi/cloud-providers/vcd/docs/ENVIRONMENT_RU.md [hunk 3]
```diff
@@ -7,52 +7,217 @@ description: "Подготовка окружения VMware Cloud Director дл
...
+Добавьте следующие правила firewall:
+
+![Настройка firewall на edge gateway, шаг 1](../../images/030-cloud-provider-vcd/edge-firewall/Screenshot5.png)
+
+## Шаблон виртуальной машины
+{% alert level="warning" %}
+Работоспособность провайдера подтверждена только для шаблонов виртуальных машин на базе Ubuntu 22.04.
+{% endalert %}
+
+В примере используется OVA-файл предоставляемый Ubuntu, с двумя исправлениями.
```
Comment (kirkonru) [inline ee/candi/cloud-providers/vcd/docs/ENVIRONMENT_RU.md:160]:
```suggestion
В примере используется OVA-файл, предоставляемый Ubuntu, с двумя исправлениями.
```

**Review by kirkonru — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Pull request merged**